packages/kotti-ui/source/kotti-filters/components/FilterRow.vue

```diff
@@ -104,11 +105,7 @@ export default defineComponent<{
 			type: String,
 		},
 		filter: {
-			default: (): KottiFilters.Filter => ({
-				key: null,
-				operation: null,
-				value: null,
-			}),
+			default: null,
```

Comment:
unrelated to the MR and goes hand in hand with other changes in this file (can you split them all outside) 

it's a side refactor that's potentially for a bug?